solr/solrj/src/java/org/apache/solr/client/solrj/request/CollectionRequiringSolrRequest.java

```diff
@@ -19,13 +19,15 @@
 import org.apache.solr.client.solrj.SolrRequest;
 import org.apache.solr.client.solrj.SolrResponse;
 
-public abstract class DataStoreSolrRequest<T extends SolrResponse> extends SolrRequest<T> {
-  public DataStoreSolrRequest(METHOD m, String path) {
+/** Parent {@link SolrRequest} class that requires a (target) collection or core. */
```

Comment:
`(target)` or just plain old `target` ?  or are you saying target is specific to a collection but NOT a core!?

Reply:
Done 👍